src/Sentry.Unity.Editor.iOS/SentryXcodeProject.cs

```diff
@@ -22,22 +22,21 @@ internal class SentryXcodeProject : IDisposable
         private readonly string _mainPath = Path.Combine("MainApp", "main.mm");
         private readonly string _optionsPath = Path.Combine("MainApp", OptionsName);
         private readonly string _uploadScript = @"
-
 process_upload_symbols() {{
-    ./{0} --log-level=info upload-dif {1} $BUILT_PRODUCTS_DIR 2>&1 | tee ./sentry-symbols-upload.log
+    ./{0} upload-dif --force-foreground {1} $BUILT_PRODUCTS_DIR 2>&1 | tee ./sentry-symbols-upload.log
 }}
 
 export SENTRY_PROPERTIES=sentry.properties
 if [ ""$ENABLE_BITCODE"" = ""NO"" ] ; then
-    echo ""note: Uploading debug symbols (Bitcode disabled).""
+    echo ""Uploading debug symbols (Bitcode disabled).""
```

Comment:
I remember `note: ` prefix having to do something with how the logs were displayed in xcode. Why removing? I may be dreaming though...

Reply:
Yes. There are `note:`, `warning:` and `error:` afaik but we don't use them for anything and since sentry cli just plain logs it messes with the order.